synapse/rest/client/v2_alpha/report_event.py

```diff
@@ -58,17 +81,108 @@ async def on_POST(self, request, room_id, event_id):
                 Codes.BAD_JSON,
             )
 
+        target = body.get("org.matrix.msc2938.target", "homeserver-admins")
+
+        if target == "room-moderators":
+            # Report event to room moderators.
+            # This branch has further safety checks (e.g. can the user actually see the event?)
+            return await self._target_room_moderators(room_id=room_id, event_id=event_id, user_id=user_id, reason=body["reason"], score=body["score"])
```

Comment:
I find the `_target` prefix unclear -- maybe `_report_to_room_moderators` and `_report_to_homeserver_admin` would be clearer?